lib/rules/no-autoplay.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @fileoverview Rule to disallow autoplay property
+ * @author Martyna Babiak
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+/** @type {import('../shared/types').Rule} */
+module.exports = {
+    meta: {
+        type: "problem",
+
+        docs: {
+            description: "Disallows autoplay property in embedded videos",
+            recommended: false,
+            url: "https://ec0lint.com/features/no-autoplay"
+        },
+        messages: {
+          error: 'Autoplay property slows down your webpage'
```

Comment:
pls change to: Autoplay property in youtube videos slows down your webpage. Delete it from the URL.

and write it in autoplay draft documentation, so that Kasia will be able to review it